### PR TITLE
Standard style

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# Combokeys
-
-[![Build Status](https://travis-ci.org/PolicyStat/combokeys.svg?branch=master)](https://travis-ci.org/mightyiam/combokeys)
+# Combokeys [![Build Status](https://travis-ci.org/PolicyStat/combokeys.svg?branch=master)](https://travis-ci.org/mightyiam/combokeys) [![js-standard-style](https://raw.githubusercontent.com/feross/standard/master/badge.png)](https://github.com/feross/standard)
```
Comment (mightyiam) [inline README.md]: This is the part I like the most. See the rich diff.

**Comment by mightyiam** [comment]
Please review.

**Comment by MitMaro** [comment]
Looks good.

**Comment by mightyiam** [comment]
Thank you, @MitMaro

**Pull request merged**